config/logging.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Default Log Channel
+    |--------------------------------------------------------------------------
+    |
+    | This option defines the default log channel that gets used when writing
+    | messages to the logs. The name specified in this option should match
+    | one of the channels defined in the "channels" configuration array.
+    |
+    */
+    'default' => env('LOG_CHANNEL', 'stack'),
```

Comment:
Should not we use `env('LOG_CHANNEL', env('APP_LOG', 'stack'))` here for backward compatibility ?